mobile/src/app/app.tsx

```diff
@@ -7,14 +7,16 @@ import { Provider as StoreProvider } from 'react-redux';
 
 import { globalStyles } from '~/bundles/common/styles/styles';
 import { store } from '~/framework/store/store';
-import { Root as RootNavigation } from '~/navigations/navigations';
+// import { Root as RootNavigation } from '~/navigations/navigations';
+import { OnboardingNavigation } from '~/navigations/onboarding-navigation/onboarding-navigation';
 
 const App: FC = () => {
     return (
         <StoreProvider store={store.instance}>
             <GestureHandlerRootView style={globalStyles.flex1}>
                 <NavigationContainer>
-                    <RootNavigation />
+                    {/* <RootNavigation /> */}
+                    <OnboardingNavigation />
```

Comment:
Let's leave root nav here, you can return OnboardingNavigation conditionally in RootNavigation

Reply:
Let's add some flag which for now we will change manually isProfileComplete (for ex.)

Reply:
See this PR:  [https://github.com/BinaryStudioAcademy/bsa-2023-bsa-talents/pull/83](https://github.com/BinaryStudioAcademy/bsa-2023-bsa-talents/pull/83)

Reply:
I am trying to do like this, but when I save prettier delets parenthesis and then liner is compaining. Maybe I should use ordinary if statement for this?

![image](https://github.com/BinaryStudioAcademy/bsa-2023-bsa-talents/assets/104503014/6f8ebab9-0bab-463a-b4d6-9c258a2c23f6)

Reply:
IsProfileComplete should be inside signed in user, firstly sign in check, then profile complete

Reply:
Add `// prettier-ignore` before return

Reply:
So, leave as I did and just add this `// prettier-ignore`?